### ISSUE TITLE
Weather icon is missing alt text

### ISSUE THREAD

**Issue by srimanh** [kalviumcommunity/devlinks-04#18]
The icon <img> in src/components/Header.jsx has no alt attribute. Add descriptive alt text.

**Comment by amityohan** [comment]
Hey I would like to fix this issue
